lib/acme/client/jwk/base.rb

```diff
@@ -0,0 +1,77 @@
+class Acme::Client::JWK::Base
```

Comment:
Any initialization args are implementation-dependent. Should I add something like this?

```ruby
def initialize
  raise NotImplementedError
end
```